### PR TITLE
Add compression and validation

### PR DESCRIPTION
**Pull request by mmaupas**

### CODE REVIEW

File: jaxili/loss.py [hunk 1]
```diff
@@ -221,3 +221,19 @@ def loss_mmd_npe(model, params, batch):
     # Compute the log-probability
     log_prob = nf(params, theta, z)
     return -jnp.mean(log_prob) + mmd_loss
+
+def loss_mse(model: Any, params: PyTree, batch: Any):
```
Comment (sachaguer) [inline jaxili/loss.py:225]: You can add the loss functions in the script `jaxili/inventory/func_dict.py`.

**Review by sachaguer — APPROVED** [review]
Looks good!
